File: src/main/java/cz/wake/manager/perks/general/Glow_command.java
package cz.wake.manager.perks.general;

import co.aikar.commands.BaseCommand;
import co.aikar.commands.CommandHelp;
import co.aikar.commands.annotation.CommandAlias;
import co.aikar.commands.annotation.Default;
import co.aikar.commands.annotation.Description;
import co.aikar.commands.annotation.HelpCommand;
import cz.craftmania.craftlibs.utils.ChatInfo;
import cz.wake.manager.Main;
import cz.wake.manager.utils.ServerType;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

@CommandAlias("glow")
@Description("Umožní ti vypnutí/zapnutí efektu glow")
public class Glow_command extends BaseCommand {

    @HelpCommand
    public void helpCommand(CommandSender sender, CommandHelp help) {
        sender.sendMessage("§e§lGlow commands:");
        help.showHelp();
    }

    @Default
    public void changeGlowEffect(CommandSender Sender) {
        if (Sender instanceof Player) {
            Player player = (Player) Sender;
            if (Main.getInstance().getServerType() == ServerType.ANARCHY) {
                ChatInfo.DANGER.send(player, "Na tomto serveru tato výhoda neplatí!");
                return;
            }
            if (player.hasPermission("craftmanager.glow")) {
                if (!player.isGlowing()) {
                    player.setGlowing(true);
                    ChatInfo.SUCCESS.send(player, "Aktivoval jsi efekt §5Glowing!");
                } else {
                    player.setGlowing(false);
                    ChatInfo.SUCCESS.send(player, "Deaktivoval jsi efekt §5Glowing!");
                }
            } else {
                ChatInfo.DANGER.send(player, "K použití této schopnosti musíš vlastnit VIP!");
            }
        }
    }
}

File: src/main/kotlin/cz/wake/manager/commands/GlowCommand.kt
package cz.wake.manager.commands

import co.aikar.commands.BaseCommand
import co.aikar.commands.CommandHelp
import co.aikar.commands.annotation.CommandAlias
import co.aikar.commands.annotation.Default
import co.aikar.commands.annotation.Description
import co.aikar.commands.annotation.HelpCommand
import cz.craftmania.craftlibs.utils.ChatInfo
import cz.wake.manager.Main
import cz.wake.manager.utils.ServerType
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import org.bukkit.potion.PotionEffectType

@CommandAlias("glow")
@Description("Umožní ti vypnutí/zapnutí efektu glow")
class GlowCommand : BaseCommand() {

    @HelpCommand
    fun helpCommand(sender: CommandSender, help: CommandHelp) {
        sender.sendMessage("§e§lGlow commands:")
        help.showHelp()
    }

    @Default
    fun changeGlowEffect(sender: CommandSender?) {
        if (sender is Player) {
            if (Main.getInstance().serverType == ServerType.ANARCHY) {
                ChatInfo.DANGER.send(sender, "Na tomto serveru tato výhoda neplatí!")
                return
            }
            if (sender.hasPermission("craftmanager.glow")) {
                if (!sender.isGlowing) {
                    sender.isGlowing = true
                    ChatInfo.SUCCESS.send(sender, "Aktivoval jsi efekt §5Glowing!")
                } else {
                    sender.isGlowing = false
                    if (sender.hasPotionEffect(PotionEffectType.INVISIBILITY)) {
                        sender.removePotionEffect(PotionEffectType.INVISIBILITY)
                    }
                    ChatInfo.SUCCESS.send(sender, "Deaktivoval jsi efekt §5Glowing!")
                }
            } else {
                ChatInfo.DANGER.send(sender, "K použití této schopnosti musíš vlastnit VIP!")
            }
        }
    }
}